2021-01-28 08:36:54,666 ERROR 5424 [-/127.0.0.1/-/104ms POST /role/add] nodejs.ER_DUP_ENTRYError: ER_DUP_ENTRY: Duplicate entry 'test' for key 'idx_roleName'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:310:20)
    at addChunk (_stream_readable.js:286:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:54:5
    sql: INSERT INTO `role`(`roleName`) VALUES('test')
code: "ER_DUP_ENTRY"
errno: 1062
sqlMessage: "Duplicate entry 'test' for key 'idx_roleName'"
sqlState: "23000"
index: 0
sql: "INSERT INTO `role`(`roleName`) VALUES('test')"
name: "ER_DUP_ENTRYError"
pid: 5424
hostname: J9MO9BU01VTBF92

2021-01-28 10:13:43,496 ERROR 20164 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:205:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 20164
hostname: J9MO9BU01VTBF92

2021-01-28 10:15:22,404 ERROR 8496 [-/127.0.0.1/-/37ms POST /role/test] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'test1' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:310:20)
    at addChunk (_stream_readable.js:286:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:54:5
    sql: SELECT  * FROM role WHERE roleName = test1 AND id = 10
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'test1' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT  * FROM role WHERE roleName = test1 AND id = 10"
name: "ER_BAD_FIELD_ERRORError"
pid: 8496
hostname: J9MO9BU01VTBF92

2021-01-28 10:47:19,996 ERROR 11340 [-/127.0.0.1/-/75ms POST /role/test] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'test' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:310:20)
    at addChunk (_stream_readable.js:286:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:54:5
    sql: SELECT  * FROM role WHERE roleName=test# AND id = '12'
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'test' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "SELECT  * FROM role WHERE roleName=test# AND id = '12'"
name: "ER_BAD_FIELD_ERRORError"
pid: 11340
hostname: J9MO9BU01VTBF92

2021-01-28 11:05:27,477 ERROR 20152 [-/127.0.0.1/-/33ms POST /role/test] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'te' or '1''' at line 1
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:310:20)
    at addChunk (_stream_readable.js:286:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:54:5
    sql: SELECT  * FROM role WHERE id = '12' AND roleName=''te' or '1''
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'te' or '1''' at line 1"
sqlState: "42000"
index: 0
sql: "SELECT  * FROM role WHERE id = '12' AND roleName=''te' or '1''"
name: "ER_PARSE_ERRORError"
pid: 20152
hostname: J9MO9BU01VTBF92

2021-01-28 11:06:02,353 ERROR 20152 [-/127.0.0.1/-/4ms POST /role/test] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'te' or 1'' at line 1
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:310:20)
    at addChunk (_stream_readable.js:286:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:54:5
    sql: SELECT  * FROM role WHERE id = '12' AND roleName=''te' or 1'
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'te' or 1'' at line 1"
sqlState: "42000"
index: 0
sql: "SELECT  * FROM role WHERE id = '12' AND roleName=''te' or 1'"
name: "ER_PARSE_ERRORError"
pid: 20152
hostname: J9MO9BU01VTBF92

2021-01-28 11:06:22,975 ERROR 20152 [-/127.0.0.1/-/3ms POST /role/test] nodejs.ER_PARSE_ERRORError: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''' at line 1
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:310:20)
    at addChunk (_stream_readable.js:286:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:54:5
    sql: SELECT  * FROM role WHERE id = '12' AND roleName='te' or 1'
code: "ER_PARSE_ERROR"
errno: 1064
sqlMessage: "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''' at line 1"
sqlState: "42000"
index: 0
sql: "SELECT  * FROM role WHERE id = '12' AND roleName='te' or 1'"
name: "ER_PARSE_ERRORError"
pid: 20152
hostname: J9MO9BU01VTBF92

2021-01-28 11:32:56,131 ERROR 20164 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:205:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 20164
hostname: J9MO9BU01VTBF92

2021-01-28 11:49:51,237 ERROR 16900 [-/127.0.0.1/-/37ms POST /role/detele] nodejs.ER_BAD_FIELD_ERRORError: ER_BAD_FIELD_ERROR: Unknown column 'roleId' in 'where clause'
    at Query.Sequence._packetToError (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Query.ErrorPacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\sequences\Query.js:79:18)
    at Protocol._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:291:23)
    at Parser._parsePacket (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:433:10)
    at Parser.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Parser.js:43:10)
    at Protocol.write (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:88:28)
    at Socket.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Connection.js:526:10)
    at Socket.emit (events.js:310:20)
    at addChunk (_stream_readable.js:286:12)
    --------------------
    at Pool.query (C:\Users\Administrator\Desktop\demo_server\node_modules\mysql\lib\Pool.js:199:23)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:29:7
    at new Promise (<anonymous>)
    at Pool.<anonymous> (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:12:10)
    at Pool.ret [as query] (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\node_modules\pify\index.js:56:34)
    at RDSClient.proto._query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\client.js:38:20)
    at RDSClient.proto.query (C:\Users\Administrator\Desktop\demo_server\node_modules\ali-rds\lib\operator.js:54:29)
    at proto.query.next (<anonymous>)
    at onFulfilled (C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:65:19)
    at C:\Users\Administrator\Desktop\demo_server\node_modules\co\index.js:54:5
    sql: DELETE FROM `role` WHERE `roleId` = 11
code: "ER_BAD_FIELD_ERROR"
errno: 1054
sqlMessage: "Unknown column 'roleId' in 'where clause'"
sqlState: "42S22"
index: 0
sql: "DELETE FROM `role` WHERE `roleId` = 11"
name: "ER_BAD_FIELD_ERRORError"
pid: 16900
hostname: J9MO9BU01VTBF92

2021-01-28 13:00:31,103 ERROR 20164 nodejs.unhandledExceptionError: read ECONNRESET
    at TCP.onStreamRead (internal/stream_base_commons.js:205:27)
errno: "ECONNRESET"
code: "ECONNRESET"
syscall: "read"
name: "unhandledExceptionError"
pid: 20164
hostname: J9MO9BU01VTBF92

2021-01-28 18:25:41,803 ERROR 20164 nodejs.ClusterClientNoResponseError: client no response in 2581837ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (C:\Users\Administrator\Desktop\demo_server\node_modules\cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 20164
hostname: J9MO9BU01VTBF92

2021-01-28 21:35:25,410 ERROR 20164 nodejs.ClusterClientNoResponseError: client no response in 10818151ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (C:\Users\Administrator\Desktop\demo_server\node_modules\cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 20164
hostname: J9MO9BU01VTBF92

2021-01-29 08:24:25,441 ERROR 20164 nodejs.ClusterClientNoResponseError: client no response in 38933111ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (C:\Users\Administrator\Desktop\demo_server\node_modules\cluster-client\lib\leader.js:77:23)
    at listOnTimeout (internal/timers.js:549:17)
    at processTimers (internal/timers.js:492:7)
name: "ClusterClientNoResponseError"
pid: 20164
hostname: J9MO9BU01VTBF92

